firebase-database/firebase-database.gradle

```diff
@@ -100,6 +100,7 @@ dependencies {
 
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.25.0'
+    testImplementation 'org.codehaus.plexus:plexus-utils:3.4.2'
```

Comment:
Out of curiosity: What are we using this library for?

Reply:
https://github.com/firebase/firebase-android-sdk/blob/ac3276ce7d5f5181b1c419731495eb88a98537af/firebase-database/src/test/java/com/google/firebase/database/PushIdGeneratorTest.java#L22 `PushIdGeneratorTest.java` uses `StringUtils` of this library to operate strings. It's not widely used. Old Robolectric versions has a dependency related to ant(I remember this one), and this dependency has direct/indirect dependency of `plexus-utils`. So previous `PushIdGeneratorTest.java` can use `StringUtils` of this library directly without any error. But newer Robolectric version removed this dependency related ant, so compiler throws an compiling error when I try to update Robolectric for firebse-database. This PR just declare this dependency explicitly to make compile and running pass. It's better to refactor `PushIdGeneratorTest.java` to remove `StringUtils` usage and `plexus-utils` library entirely.

Reply:
FYI @thatfiredev I will send a new PR to remove plexus-utils.